lib/msf/core/encoder/alphanum.rb

```diff
@@ -17,9 +17,9 @@ def initialize(info)
 
     register_options(
       [
-        OptString.new('BufferRegister', [ false, "The register that pointers to the encoded payload" ]),
-        OptInt.new('BufferOffset', [ false, "The offset to the buffer from the start of the register", off ]),
-        OptBool.new('AllowWin32SEH', [ true, "Use SEH to determine the address of the stub (Windows only)", false ])
+        OptString.new('BufferRegister', [ false, 'The register that points to the encoded payload' ]),
+        OptInt.new('BufferOffset', [ false, 'The offset to the buffer from the start of the register', off ]),
```

Comment:
I'd probably rename this to `offset`, too.

Reply:
Or just remove the variable altogether. The option is already described.

Reply:
? Not sure what you want as everything you have described so far would most likely break other modules.

Reply:
Will it?

Reply:
I think @wvu-r7 is referring to this:
```suggestion
        OptInt.new('BufferOffset', [ false, 'The offset to the buffer from the start of the register', offset ]),
```

Reply:
Yes, I'd replace it with `0` and remove the local variable above.

Reply:
@wvu-r7 To be clear did you mean changing `offset` in @adfoster-r7's example above to the value `0` instead?

Reply:
Yeah I think this what you meant, looking at the code now I see that `off` is statically defined with the value `0`. Let me fix that up for you guys so long, no need for the extra variable declaration.

Reply:
Yeah, and it also confused me, since I thought it was some weird var for `false`, but it's an integer.

Reply:
Anyway, shouldn't break anything!

Reply:
Ok should now be fixed. @wvu-r7 can you let me know if this looks good now?

Reply:
Approved.